libgraph/include/katana/PropertyGraph.h

```diff
@@ -269,13 +269,21 @@ class KATANA_EXPORT PropertyGraph {
   static Result<std::unique_ptr<PropertyGraph>> Make(
       GraphTopology&& topo_to_assign);
 
-  /// Make a property graph from topology and type arrays
+  /// [[deprecated("You should provide a rdg dir")]]
   static Result<std::unique_ptr<PropertyGraph>> Make(
```

Comment:
Can we send an electric shock to the developers who use this function?

Or send them emails about their car's extended warranty, maybe every 10 minutes?